packages/manager/apps/pci-savings-plan/public/translations/dashboard/Messages_fr_FR.json

```diff
@@ -37,5 +37,12 @@
   "dashboard_graph_y_axis_label_vcpu": "Nombre de vCPU(s)",
   "dashboard_graph_x_axis_label": "Jour",
   "dashboard_graph_included": "Inclus dans les Savings Plans",
-  "dashboard_graph_excluded": "Exclus des Savings Plans"
+  "dashboard_graph_excluded": "Exclus des Savings Plans",
+  "dashboard_resource_list_title": "Ressources utilisées",
+  "dashboard_resource_list_search_label": "Rechercher un ID",
+  "dashboard_resource_list_search_placeholder": "Rechercher un ID",
+  "dashboard_resource_list_no_results": "Aucun résultat trouvé pour \"{{searchQuery}}\"",
+  "dashboard_resource_list_no_resources": "Aucune ressource utilisée pour cette flavor",
+  "dashboard_resource_list_details": "Détails",
```

Comment:
`Détails` doesn't appear in common translations?
Other question: "Rechercher un ID" is translated twice, maybe you can make the key more generic and use it once?